decidim-core/app/helpers/decidim/widget_urls_helper.rb

```diff
@@ -2,8 +2,8 @@
 module Decidim
   module WidgetUrlsHelper
     def embed_modal_for(url)
-      js_embed_code = "#{content_tag(:script, '', src: url)}"
-      embed_code = "#{content_tag(:noscript, content_tag(:iframe, '', src: url.gsub(".js", ".html"), frameborder: 0, scrolling: "vertical"))}"
+      js_embed_code = content_tag(:script, "", src: url).to_s
```

Comment:
Yes, what I'm fearing is that `content_tag` is directly generating HTML somehow, but we need to save it to a string so that we can render it as is (not HTML safe, I mean). To be fair I missed the `to_s` the first time I saw the changes and I think it should work, but we'll add a review app somewhere and check all these changes :)